docs/ios/6x/getting-started/dsym-upload.md

```diff
@@ -33,34 +33,37 @@ You may run into an issue in Xcode 15 and above as Run Script phases now run in
 Please ensure that the upload script is the **last step** in the build phase. Placing it last ensures all necessary build artifacts are generated before the upload begins.
 :::
 
-### Script Configuration by Integration Method
+### Ensuring dSYMs are Available (Recommended)
 
-The run.sh script location depends on how you integrated the Embrace SDK:
+To prevent race conditions and ensure dSYMs are generated before upload, add these **Input Files** to your Run Script phase:
 
-#### Swift Package Manager
-
-For SPM integrations, download the support utility from [our releases page](https://github.com/embrace-io/embrace-apple-sdk/releases) and place it in your project. Then use:
-
-```bash
-EMBRACE_ID=YOUR_APP_ID EMBRACE_TOKEN=YOUR_API_TOKEN "${SRCROOT}/path/to/EmbraceIO/run.sh"
+```
+$(DWARF_DSYM_FOLDER_PATH)/$(DWARF_DSYM_FILE_NAME)/Contents/Resources/DWARF/$(PRODUCT_NAME)
+$(DWARF_DSYM_FOLDER_PATH)/$(DWARF_DSYM_FILE_NAME)
 ```
 
-#### CocoaPods
+This tells Xcode that your script depends on the dSYM files being available, ensuring proper build order and preventing the upload script from running before dSYMs are generated.
 
-For CocoaPods integrations:
+### Download and Setup the Upload Utility
 
-```bash
-EMBRACE_ID=YOUR_APP_ID EMBRACE_TOKEN=YOUR_API_TOKEN "${PODS_ROOT}/EmbraceIO/run.sh"
-```
+**Important:** Starting with iOS SDK 6.x, the dSYM upload scripts are no longer bundled with the SDK package. You must download them separately.
 
-#### Manual Integration
+1. **Download the support utility** from: https://downloads.embrace.io/embrace_support.zip
+2. **Extract the archive** and copy both `run.sh` and `upload` to a known location in your project (e.g., `Scripts/embrace/` or `third_party/embrace/`)
```

Comment:
```suggestion
2. **Extract the archive** and copy both `run.sh` and `embrace_symbol_upload.darwin` to a known location in your project (e.g., `Scripts/embrace/` or `third_party/embrace/`)
```

Reply:
"upload" is not a file, it's a symlink to embrace_symbol_upload.darwin (which I only added to be backwards-compatible with what we used to do inside embrace_support.zip long time ago)